bindings_generator/src/main.rs

```diff
@@ -1,30 +1,31 @@
+use std::path::Path;
+
 fn main() {
-    let out_path = std::env::current_dir().unwrap();
-    let header = out_path.join("wrapper.h").to_str().unwrap().to_string();
-    let out_file = out_path.join("../src/bindings.rs");
+    let header = Path::new(env!("CARGO_MANIFEST_DIR"))
+        .join("wrapper.h")
+        .to_str()
+        .unwrap()
+        .to_string();
```

Comment:
```suggestion
        .unwrap();
```

`.header()` takes `Into<String>`, which `std` provides via [`From<&str> for String>`](https://doc.rust-lang.org/std/string/struct.String.html#impl-From%3C%26str%3E-for-String).